packages/build/src/plugins_core/deploy/grpc_deploy.js

```diff
@@ -0,0 +1,78 @@
+'use strict'
+
+const { promisify } = require('util')
+
+const {
+  loadPackageDefinition,
+  credentials: { createInsecure: createCredentials },
```

Comment:
We should consider switching to TLS in a separate PR. See https://grpc.io/docs/guides/auth/#with-server-authentication-ssltls-6

We can create a certificate file only the buildbot user and `@netlify/build` user has access to.
Out of scope for this PR

Reply:
Do we need to use TLS here?
The client and the server are on the same machine. More importantly, the payload is not security sensitive at all.

Also cc'ing @vbrown608 on this.

Reply:
I don't think we need it yet, but if/when we decide to pass API tokens around we might need it (at the moment plugins and the build command can access the server too).

Reply:
Good point :+1: